### PR TITLE
change to --attach for watching apb pod

### PR DESCRIPTION
**Pull request by tchughesiv**

### CODE REVIEW

**Review by codificat — APPROVED** [review]
LGTM

File: hello-world-apb/Makefile [hunk 1]
```diff
@@ -30,11 +30,9 @@ openshift-test:
...
 	docker tag ${CONTEXT}/${IMAGE_NAME}:${TARGET}-${VERSION} ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
 	docker push ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
-	oc adm policy add-role-to-user admin -z default
-	oc run ${IMAGE_NAME} --restart=Never --image=${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME} -- provision --extra-vars 'namespace=${PROJ_RANDOM}'
-	while [ -z $${POD_STATE} ]; do sleep 2; POD_STATE=$$(oc get pod ${IMAGE_NAME} --template '{{(index .status.containerStatuses 0).state.terminated.exitCode}}' | grep -w '^[0-9]*$$'); done
-	oc describe po/${IMAGE_NAME}
-	oc logs po/${IMAGE_NAME}
+	oc create sa apb
+	oc adm policy add-role-to-user admin -z apb
+	oc run ${IMAGE_NAME} --image=${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME} --restart=Never --attach=true --overrides='{"apiVersion":"v1","spec":{"serviceAccountName":"apb"}}' -- provision -e namespace=${PROJ_RANDOM}
```
Comment (jwmatthews) [inline hello-world-apb/Makefile:35]: +1 nice to see how we can specify the service account name with oc run....very cool!  Need to get this in a FAQ so others are aware.

Comment (tchughesiv) [reply hello-world-apb/Makefile:35]: ok, want me to give it a go or? existing FAQ somewhere? maybe here to demonstrate that you don't need a `docker run` to kick off an apb?
https://github.com/fusor/apb-examples#running-the-apb

Comment (tchughesiv) [reply hello-world-apb/Makefile:35]: e.g. 
```shell
oc new-project tohughes-jenkins
oc create sa apb
oc adm policy add-role-to-user admin -z apb
oc run jenkins-apb --restart=Never --image=ansibleplaybookbundle/jenkins-apb --attach=true \
  --overrides='{"apiVersion":"v1","spec":{"serviceAccountName":"apb"}}' -- provision -e namespace=$(oc project -q)
```

**Pull request merged**